proposals/dao/fip_70.ts

```diff
@@ -0,0 +1,189 @@
+import hre, { ethers } from 'hardhat';
+import chai, { expect } from 'chai';
+import CBN from 'chai-bn';
+import { DeployUpgradeFunc, SetupUpgradeFunc, TeardownUpgradeFunc, ValidateUpgradeFunc } from '@custom-types/types';
+import { TransactionResponse } from '@ethersproject/providers';
+import { expectApproxAbs, time } from '@test/helpers';
+
+chai.use(CBN(ethers.BigNumber));
+
+let uniswapEthBalanceBefore: any;
+let balancerFeiWethPool: any;
+
+/*
+FIP-70
+DEPLOY ACTIONS:
+- Create a WeightedBalancerPoolManagerBase to be admin of the DAO's WeightedPools
+- Create a FEI 30% / WETH 70% WeightedPool
+- Create a PCVDeposit for the new B-30FEI-70WETH Balancer pool owned by the DAO
+- Create a DelayedPCVMover to move the 2nd half of WETH/FEI liquidity
+*/
+
+export const deploy: DeployUpgradeFunc = async (deployAddress, addresses, logging = false) => {
+  // Create a WeightedBalancerPoolManager contract
+  const weightedBalancerPoolManagerBaseFactory = await ethers.getContractFactory('WeightedBalancerPoolManagerBase');
+  const weightedBalancerPoolManagerBase = await weightedBalancerPoolManagerBaseFactory.deploy(addresses.core);
+  await weightedBalancerPoolManagerBase.deployTransaction.wait();
+  logging && console.log('weightedBalancerPoolManagerBase: ', weightedBalancerPoolManagerBase.address);
+
+  // Create a new FEI/WETH pool
+  const weightedPoolTwoTokensFactory = await ethers.getContractAt(
+    'IWeightedPool2TokensFactory',
+    addresses.balancerWeightedPoolFactory
+  );
+  const tx: TransactionResponse = await weightedPoolTwoTokensFactory.create(
+    'Balancer 30 FEI 70 WETH',
+    'B-30FEI-70WETH',
+    [addresses.fei, addresses.weth],
+    [ethers.constants.WeiPerEther.mul(30).div(100), ethers.constants.WeiPerEther.mul(70).div(100)], // 30% FEI 70% WETH
+    ethers.constants.WeiPerEther.mul(30).div(10_000), // 0.3% swap fees
+    true, // oracleEnabled
+    weightedBalancerPoolManagerBase.address // pool owner
+  );
+  const txReceipt = await tx.wait();
+  const { logs: rawLogs } = txReceipt;
+  balancerFeiWethPool = await ethers.getContractAt('IWeightedPool', rawLogs[0].address);
+  logging && console.log('balancerFeiWethPool address: ', balancerFeiWethPool.address);
+  logging && console.log('balancerFeiWethPool id: ', await balancerFeiWethPool.getPoolId());
+
+  // Create a new Balancer deposit for the FEI/WETH pool
+  const balancerDepositWeightedPoolFactory = await ethers.getContractFactory('BalancerPCVDepositWeightedPool');
+  const balancerDepositFeiWeth = await balancerDepositWeightedPoolFactory.deploy(
+    addresses.core,
+    addresses.balancerVault,
+    addresses.balancerRewards,
+    await balancerFeiWethPool.getPoolId(), // poolId
+    '10', // max 0.1% slippage
```

Comment:
how was 10 bps decided? seems small right?

Reply:
the slippage should be 0 because the right amount of FEI should always be minted to deposit in perfect proportions with the WETH that the contract holds. But I've set to 0.1% because the Chainlink ETH price feed may not be exactly aligned with the current Balancer pool price